### DISCUSSION TITLE
Allow ordering of filterable options in dropdown

### DISCUSSION THREAD

**Discussion by bintzandt** [laravel/nova-issues#4115] [Ideas & Feature Requests]
Nova 4 added the option of `filterable` fields. Currently, it is possible to change the query that does the filtering. However, we would also like to be able to change the ordering of the options.

For example: we have a `BelongsTo` that references a `Color`. Each color has a `name`. We would like to add `filterable` to the `BelongsTo`. The options in the dropdown are currently sorted by their ID but in this case it makes much more sense to order them by their name.

Let me know if you have any questions :)

**Comment by ninthspace** [comment]
If you implement custom ordering of a Resource as per this:

https://github.com/laravel/nova-issues/issues/156#issuecomment-962728601

The `filterable` field will be ordered.
